### PR TITLE
Updating input names and descriptions

### PR DESCRIPTION
**Pull request by shigupt202**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -14,15 +14,15 @@ inputs:
...
-  add-CIS-checks:
-    description: 'Add CIS checks to build secure and best pracitce Docker images'
+  run-quality-checks:
+    description: 'Add additional checks to ensure the image is secure and follows best practices and CIS standards'
     default: 'true'
 
 outputs:
-  scan-report:
+  scan-report-path:
     description: 'File name where the scan results are stored'
```
Comment (ajinkya599) [inline action.yaml:26]: File path?

Comment (shigupt202) [reply action.yaml:26]: Discussed with Vishal on this. This variable stores the path of scan report file, so changed it.

Comment (shigupt202) [reply action.yaml:26]: You mean the description? I'll update that.

File: src/utils.ts [hunk 2]
```diff
@@ -110,7 +110,7 @@ function getCheckText(trivyStatus: number, dockleStatus: number): string {
   const trivyText = trivyHelper.getText(trivyStatus);
   let text = trivyText;
 
-  if (inputHelper.isCisChecksEnabled()) {
+  if (inputHelper.isRunQualityChecksEnabled()) {
```
Comment (ajinkya599) [inline src/utils.ts:113]: Suggestion: `isQualityChecksEnabled`

Comment (shigupt202) [reply src/utils.ts:113]: Wanted to keep the function name consistent with the input variable name

Comment (ajinkya599) [reply src/utils.ts:113]: Both are fine. Leaving upto you.

**Pull request merged**